env.sh

```diff
@@ -45,6 +45,20 @@ action() {
         run_cmd scramv1 project CMSSW $CMSSW_VER
         run_cmd cd $CMSSW_VER/src
         run_cmd eval `scramv1 runtime -sh`
+        run_cmd git-cms-init
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/akhukhun/roccor.git RoccoR
+        run_cmd git clone git@github.com:SVfit/ClassicSVfit.git TauAnalysis/ClassicSVfit -b fastMTT_19_02_2019
+        run_cmd git clone git@github.com:SVfit/SVfitTF.git TauAnalysis/SVfitTF
```

Comment:
why do we need SVfit? shouldn't be necessary to run inference

Reply:
Scram doesn't compile without the two SVfit and the RochesterCorrection packages; it's needed by the ParticleNet setup.
As for everything else below, I commented what indeed wasn't needed